blazor/pdfviewer-2/server-deployment/deploy-maui-android.md

```diff
@@ -0,0 +1,190 @@
+
+## Adding Blazor SfPdfViewer Component
+
+Add the Syncfusion SfPdfViewer component in the **~/Pages/Index.razor** file.
+
+{% tabs %}
+{% highlight razor %}
+
+@page "/"
+
+<SfPdfViewer2 @ref="viewer" DocumentPath="@DocumentPath" Height="100%" Width="100%"></SfPdfViewer2>
+
+@code {
+    SfPdfViewer2 viewer;
+    private string DocumentPath { get; set; } = "";
+
+    protected override void OnInitialized()
+    {
+        string basePath = "MauiBlazorAndroid.wwwroot.data.pdf_succinctly.pdf";
+
+        Stream DocumentStream = this.GetType().Assembly.GetManifestResourceStream(basePath);
+        DocumentStream.Position = 0;
+        using (MemoryStream memoryStream = new MemoryStream())
+        {
+            DocumentStream.CopyTo(memoryStream);
+            byte[] bytes = memoryStream.ToArray();
+            string base64String = Convert.ToBase64String(bytes);
+            string base64prefix = "data:application/pdf;base64,";
+            //Assigned the base64 path to the PDF document path.
+            DocumentPath = $"{base64prefix}{base64String}";
+        }
+        base.OnInitialized();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> When developing a Blazor Android MAUI application, it is need to pass the `DocumentPath` to the SfPdfViewer component as a `base64 string`. This ensures that the application can retrieve and render the PDF document correctly within the SfPdfViewer.
+
+## Run the SfPdfViewer in Blazor android MAUI application
+
+To run the SfPdfViewer in a Blazor Android MAUI application using the Android emulator, follow these steps:
+
+![Run Windows machine](images/emulator.png)
+
+**step 1** Set up the necessary dependencies, SDKs, and tools for Blazor Android MAUI on your Windows machine. Ensure that you have     installed the required `Android SDK licenses`. If any errors occur during installation, follow the provided prompts or instructions to resolve them.
+
+![android-sdk-liscence](images/android-sdk-liscence.png)
+
+**step 2** Install and launch the Android Device Manager. Open the Android SDK Manager, go to the `SDK Tools` tab, select the `Android Device Manager` checkbox, and click `Apply` or `OK`. This will allow you to create, manage, and launch Android Virtual Devices (AVDs) for testing and running Android applications.
+
+![picxel emulator](images/pixcel-emulator.png)
+
+**step 3** Make sure the Android emulator is running. Launch the Android Device Manager and create or select an existing AVD to run the emulator.
+
+Now launch the project again in emulator mode it will render the SfPdfViewer using Blazor android MAUI application
+
+N> If you encounter any errors while using the Android Emulator, you can refer to the following link for troubleshooting guidance[Troubleshooting Android Emulator](https://learn.microsoft.com/en-us/dotnet/maui/android/emulator/troubleshooting)

```

Comment:
N> If you encounter any errors while using the Android Emulator, refer to the following link for troubleshooting guidance.
[Troubleshooting Android Emulator](https://learn.microsoft.com/en-us/dotnet/maui/android/emulator/troubleshooting)